src/main/asciidoc/batch/file-ingest-sftp/kubernetes.adoc

```diff
@@ -0,0 +1,278 @@
+[[sftp-file-ingest-kubernetes]]
+==== Using the Kubernetes Server
+
+===== Additional Prerequisites
+* A database tool such as link:https://dbeaver.jkiss.org/download/[DBeaver] to inspect the database contents
+* An SFTP server accessible from the Kubernetes cluster
+* An NFS server accessible from the Kubernetes cluster
+NOTE: For this example, we use an NFS host configured to allow https://www.tldp.org/HOWTO/NFS-HOWTO/server.html[read-write access].
+* Configure a Kubernetes Persistent Volume named `nfs` using the Host IP of the NFS server and the shared directory path:
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+spec:
+  capacity:
+    storage: 50Gi
+  accessModes:
+    - ReadWriteMany
+  nfs:
+    server: <NFS_SERVER_IP>
+    path: <NFS_SHARED_PATH>
+```
+Copy and save the above to `pv-nfs.yaml` and replace `<NFS_SERVER_IP>` with the IP address of the NFS Server and <NFS_SHARED_PATH> with a shared directory on the server, e.g.`/export`. Create the resource:
+```
+$kubectl apply -f pv-nfs.yaml
+persistentvolume/nfs created
+```
+* Configure a Persistent Volume Claim on the `nfs` persistent volume. We will also name the PVC `nfs`. Later, we will configure our apps to use this to mount the NFS shared directory. 
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: ""
```

Comment:
can drop this since its empty

Reply:
Yeah, if you leave it out k8s provides `storageClassName: standard`